fachhochschule-vorarlberg-author-date.csl

```diff
@@ -1,33 +1,63 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="de-DE">
   <info>
-    <title>Fachhochschule Vorarlberg (German)</title>
-    <title-short>FHV</title-short>
-    <id>http://www.zotero.org/styles/fachhochschule-vorarlberg</id>
-    <link href="http://www.zotero.org/styles/fachhochschule-vorarlberg" rel="self"/>
-    <link href="http://www.fhv.at/bibliothek/schreibzentrum/unterlagen-zum-wissenschaftlichen-arbeiten" rel="documentation"/>
+    <title>Fachhochschule Vorarlberg (author-date, German)</title>
+    <title-short>FHV Autor-Jahr</title-short>
+    <id>http://www.zotero.org/styles/fachhochschule-vorarlberg-author-date</id>
+    <link href="http://www.zotero.org/styles/fachhochschule-vorarlberg-author-date" rel="self"/>
+    <link href="https://ilias.fhv.at/goto_ilias_fhv_at_fold_176959.html" rel="documentation"/>
     <contributor>
       <name>Frank Weber</name>
-      <email>newsfranklin66@gmx.net</email>
+      <email>zotero@learningsupport.at</email>
     </contributor>
     <contributor>
       <name>Claudia Heitz</name>
       <email>frau.heitz@gmail.com</email>
     </contributor>
+    <contributor>
+      <name>Dietmar Grabher</name>
+      <email>dietmar.grabher@fhv.at</email>
+    </contributor>
+    <contributor>
+      <name>Patrick O'Brien, PhD</name>
+      <email>obrienpat86@gmail.com</email>
+    </contributor>
     <category citation-format="author-date"/>
     <category field="generic-base"/>
     <summary>Citation Style of the University of Applied Sciences Vorarlberg/Austria, based on A Harvard author-date style variant, mostly german</summary>
-    <updated>2014-09-03T00:00:00+00:00</updated>
+    <updated>2017-01-17T18:26:26+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
-    <!--  useful things: non breaking space: &#160;   -->
   </info>
+  <locale xml:lang="de">
+    <terms>
+      <term name="et-al">u.a.</term>
+      <term name="anonymous" form="short">o. A.</term>
+      <term name="no date" form="short">o. J.</term>
+      <term name="collection-editor" form="short">Hrsg.</term>
+      <term name="retrieved">abgerufen am</term>
+      <term name="composer" form="short">Komp.</term>
+      <term name="composer" form="long">Komponist</term>
+      <term name="accessed">Zugriff am</term>
+    </terms>
+  </locale>
+  <locale xml:lang="en">
+    <terms>
+      <term name="et-al">et al.</term>
+      <term name="anonymous" form="short">n. a.</term>
+      <term name="no date" form="short">n. y.</term>
+      <term name="collection-editor" form="short">Ed.</term>
+      <term name="retrieved">retrieved</term>
+      <term name="composer" form="short">Comp.</term>
+      <term name="composer" form="long">Composer</term>
+    </terms>
+  </locale>
```

Comment:
Since the style has a German "default-locale", this locale section will never be used. (@adam3smith, do you recall whether Mendeley allows the "default-locale" value to be overridden? If so, maybe that's why they added it)